### ISSUE TITLE
Fix unstable tests in unit/helper/pg-notify.js

### ISSUE THREAD

**Issue by diego-G** [LiskHQ/lisk#1068]
### Expected behaviour

Tests always pass.

### Actual behaviour

Test fails when executing the file in parallel.

```
  5 failing

  1) helpers/pg-notify
       onNotification
         should not notify about event on invalid channel, but log it:
     Uncaught TypeError: Cannot read property '0' of undefined
    at Timeout._onTimeout (test/unit/helpers/pg-notify.js:348:32)
  

  2) helpers/pg-notify
       onNotification
         should notify about round-reopened event:
     Uncaught AssertionError: expected undefined to deeply equal [ 'pg-notify: Round reopened' ]
    at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:951:19)
    at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
    at Timeout._onTimeout (test/unit/helpers/pg-notify.js:391:41)
  

  3) helpers/pg-notify
       onNotification
         should notify about round-closed event:
     Uncaught AssertionError: expected undefined to deeply equal [ Array(2) ]
    at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:951:19)
    at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
    at Timeout._onTimeout (test/unit/helpers/pg-notify.js:403:42)
  

  4) helpers/pg-notify
       onNotification
         should notify about round-reopened event (message generated by query):
     Uncaught AssertionError: expected undefined to deeply equal [ 'pg-notify: Round reopened' ]
    at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:951:19)
    at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
    at Timeout._onTimeout (test/unit/helpers/pg-notify.js:420:41)
  

  5) helpers/pg-notify
       onNotification
         should notify about round-closed event (message generated by query):
     Uncaught AssertionError: expected undefined to deeply equal [ 'pg-notify: Round closed' ]
    at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:951:19)
    at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
    at Timeout._onTimeout (test/unit/helpers/pg-notify.js:436:41)
 
Test failed: ./helpers/pg-notify.js
```

### Steps to reproduce

`npm run test-unit` || `npm run test-unit-extensive`

**Comment by diego-G** [comment]
A new intermittent failure has been occurred in a nightly build https://jenkins.lisk.io/job/lisk-core/job/1.0.0/129/. It is probably related to the timers. Log traces are here:
```
  1) helpers/pg-notify
       onNotification
         should notify about round-reopened event (message generated by query):
     Uncaught TypeError: Cannot read property '0' of undefined
    at Timeout._onTimeout (test/unit/helpers/pg-notify.js:428:32)
```

**Comment by karmacoma** [comment]
To be resolved by: https://github.com/LiskHQ/lisk/issues/1174

**Comment by karmacoma** [comment]
To be resolved by: #1174
